sparse/src/KokkosSparse_CompressedSparseMatrix.hpp

```diff
@@ -0,0 +1,1070 @@
+/*
```

Comment:
This file used to be called KokkosSparse_CrsMatrix.hpp and now defines 3 classes:
- CompressedSparseMatrix
- CrsMatrix
- CscMatrix